### PR TITLE
JAMES-2993 User Recomputing Previews Route

### PR DESCRIPTION
**Pull request by trantienduchn**

### CODE REVIEW

**Comment by chibenwa** [comment]
`WebAdminUtilsTest.serverShouldBeAbletoStartConcurrently:38 » NotTerminated`

Arf...

test this please

File: server/protocols/webadmin/webadmin-jmap/src/main/java/org/apache/james/webadmin/data/jmap/RecomputeUserFastViewProjectionItemsTask.java [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+import org.apache.james.server.task.json.dto.TaskDTOModule;
+import org.apache.james.task.Task;
+import org.apache.james.task.TaskExecutionDetails;
+import org.apache.james.task.TaskType;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import reactor.core.scheduler.Schedulers;
+
+public class RecomputeUserFastViewProjectionItemsTask implements Task {
```
Comment (remk) [inline server/protocols/webadmin/webadmin-jmap/src/main/java/org/apache/james/webadmin/data/jmap/RecomputeUserFastViewProjectionItemsTask.java]: Why discrepancy in the name between the task with  `UserFastView` and  DTOs `UserPreviews` both for the task and additional information ?

**Comment by chibenwa** [comment]
`JamesDeploymentValidationTest>DeploymentValidation.validateDeploymentWithMailsFromSmtp:95 » ConditionTimeout`

**Comment by trantienduchn** [comment]
> Maybe can add an integration tests with an user having more than one message?

I guess you meant unit tests?

**Comment by Arsnael** [comment]
```
[ERROR]   WebAdminUtilsTest.serverShouldBeAbletoStartConcurrently:38 » NotTerminated
```
? test this please

**Comment by chibenwa** [comment]
Merged

**Pull request closed without merging**